app/Components/PlayList/playlistPropsInterface/playlistPropsInterface.ts

```diff
@@ -0,0 +1,6 @@
+export interface Props {
```

Comment:
interface name should be PlaylistRowPropsInterface

file name should be playlist-row-props.interface.ts